src/components/Members Section/MembersSection.tsx

```diff
@@ -0,0 +1,14 @@
+import MemberCard from "./MemberCard"
+
+
+export default function MembersSection(data: any) {
+    return (
+        <div className="flex justify-center flex-wrap" >
+            {
+                data.map((user: any) => (
+                    <MemberCard user={user} key={user.id} />
+                ))
+            }
+        </div>

```

Comment:
this will not take data as prop, I will let it pass for now though as your task was not the integration

Reply:
Forgot to destructure, was trying to get it done quickly. Sorry about that